src/sentry/search/events/fields.py

```diff
@@ -1314,6 +1314,24 @@ def with_default(default, argument):
     return argument
 
 
+class SnQLNumericColumnNoLookup(NumericColumn):
+    measurement_aliases = {
+        MEASUREMENTS_FRAMES_SLOW_RATE,
+        MEASUREMENTS_FRAMES_FROZEN_RATE,
+        MEASUREMENTS_STALL_PERCENTAGE,
+    }
+
+    def __init__(self, name):
+        super().__init__(name)
+
+    def normalize(self, value, params):
+        super().normalize(value, params)
+        return value
+
+    def get_type(self, value):
```

Comment:
If all you are doing is calling super on get_type, you don't need this method. It will fallback to the method on the superclass

Reply:
removed!